ui/src/common/actions.ts

```diff
@@ -1596,6 +1596,16 @@ export const StateActions = {
       Array.from(state.selectedTrackIds),
       Array.from(state.selectedTrackGroupIds));
   },
+
+  setTrackShellWidth(
+    state: StateDraft,
+    args:{newWidth: number},
+  ) {
+    if (isFinite(args.newWidth)) {
```

Comment:
Finite is good (it covers those pesky NaNs), but what about zero or negative numbers? Should there be some minimum?